### PR TITLE
Style and doc fixes

### PR DESCRIPTION
**Pull request by shadowhand**

### CODE REVIEW

**Comment by shadowhand** [comment]
@ameech this one too?

File: src/Traits/JsonAwareTrait.php [hunk 1]
```diff
@@ -2,9 +2,11 @@
 
 namespace Equip\Data\Traits;
 
-trait JsonAwareTrait
+trait JsonAwareTrait /* implements JsonSerializable */
```
Comment (ameech) [inline src/Traits/JsonAwareTrait.php:5]: I don't love this, but it makes sense.

**Comment by ameech** [comment]
:+1: 

[Approved with PullApprove](https://pullapprove.com/equip/data/pull-request/5/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=equip/data)

File: src/Traits/JsonAwareTrait.php (hunk 1 above)
Comment (shadowhand) [inline src/Traits/JsonAwareTrait.php:5]: I wish traits could implement interfaces. I like knowing how the trait should be used and this is the best I've come up with.

Comment (ameech) [inline src/Traits/JsonAwareTrait.php:5]: Me too and this works better then just adding regular documentation.

**Pull request merged**